### PR TITLE
Add TDP 42 & Update Headers

### PR DESCRIPTION
**Pull request by mzero2nine**
Hi @dimsome san,

could you please review TDP 42 & the changes to the headers made? 

Thank you and have a great day! 😊

### CODE REVIEW

File: TDP/tdp-42.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+1,000,000 mUSD would then be transferred to the mStable Funding subDAO. 500,000 mUSD would be used to immediately pay back the loan to the investor, and the remaining 500,000 mUSD would be used to replenish the pre-approved project funding.
+
+The 1,000,000 mUSD loan to KBIT can be recalled within 48 hours notice if the funds are required for other uses and/or the decision is made to stop supporting mUSD liquidity on Coinbase.
+
+As part of this proposal, it is also suggested that the TreasuryDAO be given discretionary power to recall the loan and delist mUSD if the benefits are no longer seen to outweigh the opportunity cost and risk. This will avoid the need for another full governance proposal in the case that this is deemed appropriate, and therefore may reduce operational workload in the future.
+
+## Next Steps
```
Comment (dimsome) [inline TDP/tdp-42.md:50]: please remove the next steps from the official proposal, only add to the forum post.

**Review by dimsome — CHANGES_REQUESTED** [review]
one smoll thing

File: TDP/tdp-42.md (hunk 1 above)
Comment (mzero2nine) [reply TDP/tdp-42.md]: Absolutely, sorry I should have proofread it once again before publishing it from the received file. Lesson learned, changes committed, and feedback will be forwarded for the future :)

Comment (dimsome) [reply TDP/tdp-42.md]: nooooo issues ser

**Review by dimsome — APPROVED** [review]
lgtm

**Pull request merged**